PyISY/configuration.py

```diff
@@ -1,10 +1,13 @@
+"""ISY Configuration Lookup."""
 from xml.dom import minidom
```

Comment:
Is this still needed?

Reply:
Is `minidom` still needed, or the whole configuration file? `minidom` is still used in the `parse` function of this file.  As far as I can tell, the `configuration` lookup is used to check if the climate and networking modules are enabled.

Reply:
👍